pylintrc

```diff
@@ -59,7 +59,7 @@ disable=invalid-name,protected-access,fixme,too-many-branches,
         import-error,too-many-lines,too-many-instance-attributes,
         too-many-public-methods,duplicate-code,broad-except,
         redefined-builtin,anomalous-backslash-in-string,
-        missing-docstring,no-member,
+        missing-docstring,no-member,overlapping-except,
```

Comment:
One issue with this check is when linting code that is supposed to run both on e.g. Python 2 and Python 3. Can it be made disabled by default somehow?